services/ai-navigator-app/0.1.0/ai-navigator-app.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ai-navigator-app
+  namespace: ${releaseNamespace}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ai-navigator-app
+  namespace: ${releaseNamespace}
+rules:
+  - apiGroups: ["kommander.mesosphere.io"]
+    resources: ["licenses"]
+    verbs: ["get", "list"]
+  - apiGroups: [""]
+    resources: ["secrets"]
+    verbs: ["get"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ai-navigator-app
+  namespace: ${releaseNamespace}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ai-navigator-app
+subjects:
+  - kind: ServiceAccount
+    name: ai-navigator-app
+    namespace: ${releaseNamespace}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ai-navigator-app-config
+  namespace: ${releaseNamespace}
+data:
+  CHATBOT_URL: "https://chatbot.api.d2iq.com/v1/chat"
+  AUTHORIZER_URL: "https://auth.api.d2iq.com"
+  LICENSE_NAMESPACE: ${releaseNamespace}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ai-navigator-app
+  namespace: ${releaseNamespace}
+  labels:
+    app: ai-navigator-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ai-navigator-app
+  template:
+    metadata:
+      labels:
+        app: ai-navigator-app
+    spec:
+      serviceAccountName: ai-navigator-app
+      containers:
+        - name: ai-navigator-app
+          image: mesosphere/ai-navigator-app:development # TODO: Version
```

Comment:
Do you want to land it once there is a version or earlier?

We could progress with even before the `ai-navigator-app` first release - in that case, let's just make it an empty customization (no deployments, etc).

Reply:
I'm not sure what advantage we have when landing this without the resources?
Also, we still need to get the correct icon and description.

I would argue that we should create dev releases regularly so it's easy to test this service. 
What do you think?

Reply:
We do not want regular releases.